tutorials/rendering/renderers.rst

```diff
@@ -148,8 +148,9 @@ Overall comparison
 |                     |                          |                          | optimized. Use Mobile or |
 |                     |                          |                          | Compatibility instead.   |
 +---------------------+--------------------------+--------------------------+--------------------------+
-| XR                  | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
-|                     |                          |                          |                          |
+| XR                  | ✔️ Yes. Recommended for  | ✔️ Yes. Recommended for | ⚠️ Supported, but poorly |
+|                     | stand-alone headsets     |  desktop headsets        | optimized. Use Mobile or  |
+|                     |                          |                          | Compatibility instead.   |
```

Comment:
```suggestion
| XR                  | ✔️ Yes. Recommended for  | ✔️ Yes. Recommended for  | ⚠️ Supported, but poorly |
|                     | stand-alone headsets     |  desktop headsets        | optimized. Use Mobile or |
|                     |                          |                          | Compatibility instead.   |
```
The tables are misaligned. It's because emojis are annoyingly not monospaced, but a space and a half. I did my best guess to fix it with the web UI